src/api/faq/faq.model.js

```diff
@@ -1 +1,21 @@
+const mongoose = require('mongoose');
 
+const { Schema } = mongoose;
+
+const FAQSchema = new Schema({
+  question: {
+    type: String,
+    required: true,
+  },
+  answer: {
+    type: String,
+    required: true,
+  },
+  dateUpdated: {
```

Comment:
Lets use the standard `createdAt` and `updatedAt` fields. Mongoose can automatically update these.

https://mongoosejs.com/docs/guide.html#timestamps